tools/docker/builder/openwrt/build.sh

```diff
@@ -20,13 +21,18 @@ usage() {
 }
 
 build_image() {
+    if [ -z "$1" ] ; then
+	ARGS="$scriptdir/"
+    else
+	ARGS="$* $scriptdir/"
+    fi
     # We first need to build the corresponding images
     docker build --tag "$image_tag" \
            --build-arg OPENWRT_REPOSITORY \
            --build-arg OPENWRT_VERSION \
            --build-arg TARGET \
            --build-arg PRPL_FEED \
-           "$scriptdir/"
+           "$ARGS"
```

Comment:
This isn't going to work: the whole thing will be passed as a single argument with an embedded space and won't be recognized as an option.

I changed this to instead use the BUILD_OPTIONS directly, unquoted (so if multiple options are provided they are properly passed as individual options).